processing/src/main/java/io/druid/query/groupby/GroupByQuery.java

```diff
@@ -120,12 +120,15 @@ public GroupByQuery(
       Preconditions.checkArgument(spec != null, "dimensions has null DimensionSpec");
     }
     this.aggregatorSpecs = aggregatorSpecs == null ? ImmutableList.<AggregatorFactory>of() : aggregatorSpecs;
-    this.postAggregatorSpecs = postAggregatorSpecs == null ? ImmutableList.<PostAggregator>of() : postAggregatorSpecs;
+    this.postAggregatorSpecs = Queries.prepareAggregations(
```

Comment:
I meant, Queries.prepareAggregations could do this… return an empty list if postAggregatorSpecs is null. This is minor thing and not really necessary though.